libavfilter/dnn_filter_common.h

```diff
@@ -30,7 +30,9 @@ typedef struct DnnContext {
     char *model_filename;
     DNNBackendType backend_type;
     char *model_inputname;
-    char *model_outputname;
+    char **model_outputnames;
```

Comment:
how about put model_outputnames before dnn_modules?

Reply:
i'm confused. Isn't it better to put the related struct member together?

Reply:
options together, non-options together